src/components/calendar/calendar.ts

```diff
@@ -105,6 +106,7 @@ export const focusActiveDate = Symbol();
  * @csspart selected - Indicates selected state. Applies to date, month and year elements of the calendar.
  * @csspart current - Indicates current state. Applies to date, month and year elements of the calendar.
  */
+@blazorIndirectRender
```

Comment:
This sould also work with the comment tag, no? Like `/* blazorIndirectRender */`?

Reply:
It shouldn't :) There's a diff with `/* not a doc comment */`  and `/** doc comment */` + those can be mixed like we do in Angular (see [action strip](https://github.com/IgniteUI/igniteui-angular/blob/88d98ab0e0b0fed08f27f1d60d38b6da1fdf4140/projects/igniteui-angular/src/lib/action-strip/action-strip.component.ts#L45) for example).